tst/EnergyPlus/unit/DataSurfaces.unit.cc

```diff
@@ -84,6 +84,349 @@ using namespace EnergyPlus::SurfaceGeometry;
 using namespace ObjexxFCL;
 using DataVectorTypes::Vector;
 
+TEST_F( EnergyPlusFixture, DataSurfaces_ProcessSurfaceVertices )
```

Comment:
@rraustad Since ProcessSurfaceVertices is in SurfaceGeometry, this unit test should be there as well, next to the current one which exercises ProcessSurfaceVertices in a different way.  Good to have both forms.